crawl-ref/source/mon-place.cc

```diff
@@ -304,31 +264,14 @@ static void _apply_ood(level_id &place)
         }
     }
 
-    // On D:13 and deeper, and for those who tarry, something extreme:
-    if (env.turns_on_level > 1400 - place.absdepth() * 117
-        && x_chance_in_y(_scale_spawn_parameter(2, 10000, 10000, 3000, 9000),
-                         10000))
-    {
-        // this maxes depth most of the time
-        place.depth += random2avg(27, 2);
-        dprf(DIAG_MONPLACE, "Super OOD roll: Old: %s, New: %s",
-             old_place.describe().c_str(), place.describe().c_str());
-    }
-}
-
-static int _vestibule_spawn_rate()
-{
-    // Monster generation in the Vestibule drops off quickly.
-    const int taper_off_turn = 500;
-    int genodds = 240;
-    // genodds increases once you've spent more than 500 turns in Hell.
-    if (env.turns_on_level > taper_off_turn)
-    {
-        genodds += (env.turns_on_level - taper_off_turn);
-        genodds  = (genodds < 0 ? 20000 : min(genodds, 20000));
-    }
-
-    return genodds;
+     // On D:13 and deeper, something extreme:
+     if (place.absdepth() >= 12 && x_chance_in_y(2, 10000))
```

Comment:
Why 2/10000 instead of the simpler 1/5000?

Reply:
This could be simplified as you say, but elliptic and I agreed that it can simply be removed. It's a very low chance for monsters to be "super OOD" for absdepth equivalent to D:13 and later. It was more meaningful when it was scaled by turns, but vaults placing OOD through `8` and `9` monsters is already common in these later levels. Will remove this block when I rebase/merge.